tests/iio_common.c

```diff
@@ -434,6 +440,19 @@ void usage(char *name, const struct option *options,
 	exit(0);
 }
 
+void version(char *name)
+{
+	unsigned int i, major, minor;
+        char git_tag[8];
+
+	printf("%s version: %u.%u (git tag:%s)\n", name, LIBIIO_VERSION_MAJOR, LIBIIO_VERSION_MINOR, LIBIIO_VERSION_GIT);
+	iio_library_get_version(&major, &minor, git_tag);
+        printf("libiio version: %u.%u (git tag: %s) backends:", major, minor, git_tag);
```

Comment:
`libiio` -> `Libiio`? :)

Reply:
sure - I thought that you or Lars always left it lower case on purpose...